Userland/Libraries/LibPDF/Fonts/Type3Font.cpp

```diff
@@ -28,8 +53,25 @@ void Type3Font::set_font_size(float)
 {
 }
 
-PDFErrorOr<void> Type3Font::draw_glyph(Gfx::Painter&, Gfx::FloatPoint, float, u8, Color)
+PDFErrorOr<void> Type3Font::draw_glyph(Gfx::Painter&, Gfx::FloatPoint, float, u8 char_code, Color)
 {
+    // "For each character code shown by a text-showing operator that uses a Type 3 font,
+    //  the consumer application does the following:""
```

Comment:
Micronit-pick: Doubled " at the end here?

Reply:
Done.